swarmrl/engine/espresso.py

```diff
@@ -48,6 +52,7 @@ class to hold all information needed to setup and run the MD simulation.
     time_step: pint.Quantity
     time_slice: pint.Quantity
     write_interval: pint.Quantity
```

Comment:
I guess periodicity should also go in here. We can save this for a different PR.